stable/traefik/README.md

```diff
@@ -114,11 +114,26 @@ $ helm install --name my-release --namespace kube-system --values values.yaml st
 
 ### Clustering / High Availability
 
-Currently it is possible to specify the number of `replicas` but the implementation is naive.
+Currently it is possible to configure this chart to run multiple Traefik instances using
+either the `replicas` option or the `daemonset.enabled` option.  Enabling either results
+in a naive Traefik implementation.
 
 **Full Traefik clustering with leader election is not yet supported.**
 
-It is heavily advised to not set a value for `replicas` if you also have Let's Encrypt configured. While setting `replicas` will work for many cases, since no leader is elected it has the consequence that each node will end up requesting Let's Encrypt certificates if this is also configured. This will quickly cut into the very modest rate limit that Let's Encrypt enforces.
+Specifically, it is heavily advised to not enable 'daemonset.enabled` or set a value
+for `replicas` if you also have Let's Encrypt configured. While setting multiple 
+Traefik instances will work for many cases, since no leader is elected it has the consequence 
+that each node will end up requesting Let's Encrypt certificates if this is also configured.
+This will quickly cut into the very modest rate limit that Let's Encrypt enforces. 
+
+If you are installing the chart to a cluster that is not capable of provisioning an
+external load balancer (i.e. not using AWS or GKE), using `daemonset.enabled=true`
+can help make Traefik accessible outside of the cluster on common 80/443 ports.
+Configuring a DaemonSet will schedule a Traefik pod on each node with `hostNetwork` enabled 
+due to https://github.com/kubernetes/kubernetes/issues/23920.  You will likely also want
+to change the `dashboard.port` so it doesn't conflict with your master node's
```

Comment:
So... I have already registered some concern over `dashboard.port`. I see, now, after reading this bit of doc change that what you've implemented isn't in line with what you've documented here. What you've implemented gives control over the port that the dashboard listens on within the container... that's not the knob you want to expose. You want to control the port the _service_ listens on when a daemonset is used.